spec/support/shared/examples/metasploit/cache/module/rankable.rb

```diff
@@ -0,0 +1,9 @@
+shared_examples_for 'Metasploit::Cache::Module::Rankable' do |rank:|
+  context 'associations' do
+    it { is_expected.to belong_to(:rank).class_name('Metasploit::Cache::Module::Rank').inverse_of(rank.fetch(:inverse_of) ) }
```

Comment:
Line is too long. [125/80]<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.